modular_skyrat/modules/emotes/code/emotes.dm

```diff
@@ -0,0 +1,315 @@
+					to_chat(S, "<span class='warning'>Scream module deactivated. Please recharge.</span>")
+					return
+				S.cell.use(200)
+		if(ismonkey(user))
+			sound = 'modular_skyrat/modules/emotes/sound/voice/scream_monkey.ogg'
+		if(istype(user, /mob/living/simple_animal/hostile/gorilla))
+			sound = 'sound/creatures/gorilla.ogg'
+		if(ishuman(user))
+			user.adjustOxyLoss(5)
+			var/mob/living/carbon/human/H = user
+			var/datum/species/userspecies = H.dna.species
+			if(H)
+				if(userspecies.screamsounds.len)
+					sound = pick(userspecies.screamsounds)
+				if(H.gender == FEMALE)
+					if(userspecies.femalescreamsounds.len)
+						sound = pick(userspecies.femalescreamsounds)
+		if(isalien(user))
+			sound = 'sound/voice/hiss6.ogg'
+		/* TO DO - REINDEX SCREAMS
+		LAZYINITLIST(user.alternate_screams)
+		if(LAZYLEN(user.alternate_screams))
+			sound = pick(user.alternate_screams)
+		*/
+		playsound(user.loc, sound, 50, 1, 4, 1.2)
+		message = "screams!"
+	else if(miming)
+		message = "acts out a scream."
+	else
+		message = "makes a very loud noise."
+	. = ..()
+
+/datum/emote/living/cough/run_emote(mob/living/user, params)
+	if(!(. = ..()))
+		return
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	if (isvox(user))
+		playsound(user, 'modular_skyrat/modules/emotes/sound/emotes/voxcough.ogg', 50, 1, -1)
+
+/datum/emote/living/sneeze/run_emote(mob/living/user, params)
+	if(!(. = ..()))
+		return
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	if (isvox(user))
+		playsound(user, 'modular_skyrat/modules/emotes/sound/emotes/voxsneeze.ogg', 50, 1, -1)
+
+/datum/emote/living/peep
+	key = "peep"
+	key_third_person = "peeps like a bird"
+	message = "peeps like a bird!"
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = FALSE
+	restraint_check = FALSE
+	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
+
+/datum/emote/living/peep/run_emote(mob/living/user, params)
+	if(!(. = ..()))
+		return
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	playsound(user, 'modular_skyrat/modules/emotes/sound/voice/peep_once.ogg', 50, 1, -1)
+
+/datum/emote/living/peep2
+	key = "peep2"
+	key_third_person = "peeps twice like a bird"
+	message = "peeps twice like a bird!"
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = FALSE
+	restraint_check = FALSE
+	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
+
+/datum/emote/living/peep2/run_emote(mob/living/user, params)
+	if(!(. = ..()))
+		return
+	if(user.nextsoundemote >= world.time)
+		return
+	user.nextsoundemote = world.time + 7
+	playsound(user, 'modular_skyrat/modules/emotes/sound/voice/peep.ogg', 50, 1, -1)
+
+/mob
+	var/nextsoundemote = 1
+
+//Disables the custom emote blacklist from TG that normally applies to slimes.
+/datum/emote/living/custom
+	mob_type_blacklist_typecache = list(/mob/living/brain)
+
+/datum/emote/living/insult
```

Comment:
Remove this one and the insult JSON file? We really don't need this one.

Reply:
Yeah agreed.